.nvmrc

```diff
@@ -0,0 +1 @@
+14
```

Comment:
npm versions higher than 6 (which is what comes with node 14) change the version of package-lock.json from 1 to 2.

Version 2 is backwards compatible with package-lock.json version 1, but I didn't want to make that change in this PR.